Anlab.Mvc/Views/Payment/Pay.cshtml

```diff
@@ -0,0 +1,53 @@
+@model AnlabMvc.Models.Order.OrderReviewModel
+
+@{
+    ViewData["Title"] = "Pay";
+    var paymentDict = (Dictionary<string, string>)ViewBag.PaymentDictionary;
+    ViewBag.OrderId = Model.Order.Id;
+}
+<h2>Details</h2>
+
+
+@Html.Partial("_OrderDetails")
+
+
+<form action="@ViewBag.CyberSourceUrl" method="post" autocomplete="off">
+    @foreach (var pair in paymentDict)
+    {
+        <input type="hidden" name="@pair.Key" value="@pair.Value" />
+    }
+    <input type="hidden" name="signature" value="@ViewBag.Signature" />
+    <button type="submit" class="btn btn-primary"><i class="fa fa-credit-card" aria-hidden="true"> @(String.Format("{0:C}", Model.OrderDetails.GrandTotal)) Pay Now</i></button>
+</form>
+
+@*<div>
+    <h2>Debugging Info</h2>
+    
+    <div>
+        @foreach (var key in paymentDict)
+        {
+            <p>@key.Key -- @key.Value</p>
+        }
+    </div>
+</div>*@
+
+@section AdditionalStyles
+{
+    @{await Html.RenderPartialAsync("_DataTableStylePartial"); }
+}
+
+@section Scripts
+{
+    @{await Html.RenderPartialAsync("_DataTableScriptsPartial"); }
+
+    <script type="text/javascript">
+        $(function() {
+            $("#table").dataTable();
+        });
+    </script>   
+
+    @{await Html.RenderPartialAsync("_ShowdownScriptsPartial"); }

```

Comment:
@jSylvestre why does the receipt page need the showdown scripts?

Reply:
@srkirkland I'm showing the Order details partial view. It shows the test item test notes.